### DISCUSSION TITLE
If content is hidden, should it still appear in the sitemap ?

### DISCUSSION THREAD

**Discussion by jehoshua7** [McShelby/hugo-theme-relearn#68] [Q&A] [answered]
I have been using the following code to hide content

`hidden = true`

which works fine, yet I noticed the content still appears in the sitemap. Should the **hidden** feature extend to the sitemap ?

Ref - [Sitemap template/](https://gohugo.io/templates/sitemap-template/) and [ Means to exclude content from sitemap.xml #653 ](https://github.com/gohugoio/hugo/issues/653)

**Comment by jehoshua7** [comment]
I have had a reply on the Hugo discourse forums [Sitemap content and the ‘hidden’ feature](https://discourse.gohugo.io/t/sitemap-content-and-the-hidden-feature/34486/4)

I assume the necessary code would be similar to line 108 in /layouts/partials/menu.html

`{{ $hidden := and (or (.Params.hidden) (eq .Title "")) (not $.showhidden) }}`

As there is no sitemap template at present, is the template at https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/_default/sitemap.xml suitable ?

**Comment by McShelby** [comment, marked as answer]
I've made an issue for this #85. This is now fixed.
